lib/tests/streamlit/script_request_queue_test.py

```diff
@@ -83,13 +83,13 @@ def test_rerun_data_coalescing(self):
         _create_widget("trigger", states).trigger_value = True
         _create_widget("int", states).int_value = 123
 
-        queue.enqueue(ScriptRequest.RERUN, RerunData(argv=None, widget_state=states))
+        queue.enqueue(ScriptRequest.RERUN, RerunData(widget_state=states))
```

Comment:
Asking a question out of ignorance here, what happened with RerunData?

Reply:
Haha. It's a great question. The answer started as tests were not passing. I'm _guessing_ that pytest does not run tests with the `_test.py` as a suffix of the file name. This refactor makes this test _actually_ run, which if failed cause `argv` was not valid.

Turns out, it was [removed here](https://github.com/streamlit/streamlit/commit/3468fedca4f30580e0ed12c6a87bc5602c2da25d#diff-8314db9cc7432a9851d435832811aea0L39) and just tests did not fail cause they were never ran. Fortunately, removing the `argv` param made the tests pass again.